### PR TITLE
[DONE] Update content/community/nav.yml

### PR DESCRIPTION
**Pull request by Aissaoui-Ahmed**
hi everyone ! 

translate file `content/community/nav.yml`

### CODE REVIEW

File: content/community/nav.yml [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
   items:
     - id: debugging-tools
-      title: Debugging
+      title: التنقيح
     - id: component-workbenches
-      title: Component Workbenches
+      title: مكون صف العمل
     - id: jsx-integrations
-      title: JSX Integrations
+      title: دمج JSX
```
[the remark this answers is not in the document]
Comment (Aissaoui-Ahmed) [reply content/community/nav.yml]: إضافات == add-ons

File: content/community/nav.yml [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
+      title: إدارة النماذج
     - id: data-fetching
-      title: Data Fetching
+      title: جلب البيانات
     - id: testing
-      title: Testing
+      title: تجريب
     - id: ui-components
-      title: UI Components
+      title: مكونات UI
```
[the remark this answers is not in the document]
Comment (Aissaoui-Ahmed) [reply content/community/nav.yml]: In the glossary "Content that doesn't need translation"
UI

File: content/community/nav.yml [hunk 3]
```diff
@@ -1,42 +1,42 @@
...
+      title: الفيديوهات
     - id: external-resources
-      title: External Resources
-- title: أدوات
+      title: المصادر الخارجية
+- title: الأدوات
   items:
     - id: debugging-tools
-      title: Debugging
+      title: التنقيح
```
Comment (3imed-jaberi) [inline content/community/nav.yml]: للتوضيح اكثر --> `التنقيح البرمجة` ..

File: content/community/nav.yml [hunk 4]
```diff
@@ -1,42 +1,42 @@
...
+      title: دمج JSX
     - id: starter-kits
-      title: Starter Kits
+      title: أدوات البداية
     - id: routing
-      title: Routing
+      title: المسارات
     - id: model-management
-      title: Model Management
+      title: إدارة النماذج
```
Comment (3imed-jaberi) [inline content/community/nav.yml]: `نموذج` ✔️
`النماذج` ❌ 

@Fcmam5  @mohshbool

File: content/community/nav.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Fcmam5) [reply content/community/nav.yml]: The section has some integrations => plugins/addons for the JSX and doesn't discuss how to integrate/merge JSX, please review its context here: https://reactjs.org/community/jsx-integrations.html

File: content/community/nav.yml (hunk 4 above)
Comment (Fcmam5) [reply content/community/nav.yml]: @Aissaoui-Ahmed's translation (`إدارة النماذج`) is correct

File: content/community/nav.yml [hunk 5]
```diff
@@ -1,42 +1,42 @@
...
-      title: External Resources
+      title: المصادر الخارجية
 - title: أدوات
   items:
     - id: debugging-tools
-      title: Debugging
+      title: التنقيح
     - id: component-workbenches
-      title: Component Workbenches
+      title: مكون صف العمل
```
[the remark this answers is not in the document]
Comment (Fcmam5) [reply content/community/nav.yml]: > Please review this translation
> ```suggestion
>       title: منصّات تطوير المكوّنات
> ```
@Aissaoui-Ahmed please review this translation! 
ping @mohshbool @iRayan7

File: content/community/nav.yml (hunk 4 above)
Comment (3imed-jaberi) [reply content/community/nav.yml]: it's `Model` not `Models`  ??!! ..

Comment (Fcmam5) [reply content/community/nav.yml]: No need to translate words `حرفيًّا`.
Also, management is a [collective noun](https://en.wikipedia.org/wiki/Collective_noun), please refere to [this answer](https://english.stackexchange.com/a/46610).
Example:
- Project management ([Wiki](https://en.wikipedia.org/wiki/Project_management)) => إدارة المشاريع ([Wiki](https://ar.wikipedia.org/wiki/%D8%A5%D8%AF%D8%A7%D8%B1%D8%A9_%D8%A7%D9%84%D9%85%D8%B4%D8%A7%D8%B1%D9%8A%D8%B9))

File: content/community/nav.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (3imed-jaberi) [reply content/community/nav.yml]: I think the word `التكامل مع JSX` or `تكامل JSX` is better ..
1. We have translated the word previously . [look](https://ar.reactjs.org/docs/integrating-with-other-libraries.html)
2. When checking page content, you find other libraries and tools, where the word `تكامل` becomes more expressive ..

File: content/community/nav.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (3imed-jaberi) [reply content/community/nav.yml]: I think the word `التكامل` or `تكامل` is better ...

1. We have translated the word previously.  [look here](https://ar.reactjs.org/docs/integrating-with-other-libraries.html)
2. When checking page content, you find other libraries and tools. where the word `تكامل` becomes more expressive.

I don't know why @Fcmam5 loves the plural format 🤣 🤣  though it's not necessary but no problem ..

File: content/community/nav.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Fcmam5) [reply content/community/nav.yml]: The context is not the same, in that section, you meant how to integrate (as combine or fuse) with other libraries, but [here](https://reactjs.org/community/jsx-integrations.html) we are talking about plugins and add-ons

File: content/community/nav.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (Fcmam5) [reply content/community/nav.yml]: The problem is not in that, but in `صف العمل`, Workbenches means `منضدة` or `منصّة`. For you, what does `صف العمل` means?

File: content/community/nav.yml [hunk 6]
```diff
@@ -1,42 +1,42 @@
...
+      title: الدعم
     - id: courses
-      title: Courses
+      title: الدروس
     - id: examples
-      title: Examples
+      title: الأمثلة
     - id: meetups
-      title: Meetups
+      title: القاءات
```
Comment (Fcmam5) [inline content/community/nav.yml]:
```suggestion
      title: اللقاءات
```

File: content/community/nav.yml [hunk 7]
```diff
@@ -1,42 +1,42 @@
...
+      title: الفيديوهات
     - id: external-resources
-      title: External Resources
-- title: أدوات
+      title: المصادر الخارجية
+- title: الأدوات
   items:
     - id: debugging-tools
-      title: Debugging
+      title: التنقيح البرمجة
```
Comment (Fcmam5) [inline content/community/nav.yml]: I think that only `التنقيح` word is enough and clear

```suggestion
      title: التنقيح
```

Comment (3imed-jaberi) [reply content/community/nav.yml]: التنقيح البرمجي is better !

Comment (Fcmam5) [reply content/community/nav.yml]: @3imed-jaberi yes, please edit your comment and add it as a suggestion so Ahmed can commit it directly :)

File: content/community/nav.yml (hunk 6 above)
Comment (3imed-jaberi) [reply content/community/nav.yml]:
```suggestion
      title: المُلْتَقَيات 
```

Comment (3imed-jaberi) [reply content/community/nav.yml]:
```suggestion
      title: المُلْتَقَيات 
```

**Comment by Aissaoui-Ahmed** [comment]
done!

File: content/community/nav.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (3imed-jaberi) [reply content/community/nav.yml]: i'm confused about this but ....

[the remark this answers is not in the document]
Comment (Aissaoui-Ahmed) [reply content/community/nav.yml]: JSX not extension to add-on React is tag syntax .

[the remark this answers is not in the document]
Comment (3imed-jaberi) [reply content/community/nav.yml]: i khnow , i'm confused about `إضافات ` ...

**Pull request merged**